src/entity/posts/Post.ts

```diff
@@ -196,4 +198,9 @@ export class Post {
   @Column({ type: 'uuid', nullable: true })
   @Index('IDX_yggdrasil_id')
   yggdrasilId: string;
+
+  @OneToMany(() => PostRelation, (postRelation) => postRelation.post, {
```

Comment:
Is it going to be oneToMany?
Wondering if manyToMany would be better, not sure if for instance we have recommended or something later which could link post-post-post

Reply:
It is many to many but `@ManyToMany` in graphql means it will create and manage a table like `PostRelation` by itself and we want to have control over it with our own entity. Basically [this](https://orkhan.gitbook.io/typeorm/docs/many-to-many-relations#many-to-many-relations-with-custom-properties)

Reply:
Got it thanks for the context :)